glean-core/android/src/main/java/mozilla/telemetry/glean/rust/LibGleanFFI.kt

```diff
@@ -15,6 +15,9 @@ import java.lang.reflect.Proxy
 // Turn a boolean into its Byte (u8) representation
 internal fun Boolean.toByte(): Byte = if (this) 1 else 0
 
+// Turn a Byte into a boolean where zero is false and non-zero is true
+internal fun Byte.toBoolean(): Boolean = if (this == 0.toByte()) false else true
```

Comment:
I'm fine with either way, but it could avoid the if with:

```suggestion
internal fun Byte.toBoolean(): Boolean = this != 0.toByte()
```

Reply:
Oh, dear.  Of course.  That's better.